renovate.json

```diff
@@ -48,6 +49,18 @@
       "depTypeList": [
         "dependencies"
       ],
+      "updateTypes": ["major", "minor"],
+      "rangeStrategy": "auto"
+    },
+    {
+      "description": "Match our own packages where patch versions need consistency between apps, like for UI. The last match rule takes precident so this needs to be below our generic internal rule",
+      "packageNames": [
+        "@financial-times/n-myft-ui"
```

Comment:
Open to suggestions on any other very UI specific packages which we always want to keep consistency across our estate.

Reply:
Do we want to add Snyk to this?

Reply:
@edds x-dash and Page Kit should be included in this (packages that start with `@financial-times/x-` and `@financial-times/dotcom-`